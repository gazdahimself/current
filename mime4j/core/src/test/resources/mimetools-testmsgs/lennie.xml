<message>
<header>
<field>
Return-Path: &lt;lbj_ccsi@atl.mindspring.com&gt;</field>
<field>
Received: from brickbat8.mindspring.com (brickbat8.mindspring.com [207.69.200.11])
        by camel10.mindspring.com (8.8.5/8.8.5) with ESMTP id GAA27894
        for &lt;lbj_ccsi@atl.mindspring.com&gt;; Fri, 25 Jul 1997 06:58:07 -0400 (EDT)</field>
<field>
Received: from lennie (user-2k7i8oq.dialup.mindspring.com [168.121.35.26])
        by brickbat8.mindspring.com (8.8.5/8.8.5) with SMTP id GAA22488
        for &lt;lbj_ccsi@atl.mindspring.com&gt;; Fri, 25 Jul 1997 06:58:05 -0400 (EDT)</field>
<field>
Message-ID: &lt;33D89532.29EA@atl.mindspring.com&gt;</field>
<field>
Date: Fri, 25 Jul 1997 06:59:46 -0500</field>
<field>
From: Lennie Jarratt &lt;lbj_ccsi@mindspring.com&gt;</field>
<field>
Reply-To: lbj_ccsi@mindspring.com</field>
<field>
Organization: Custom Computer Services Inc.</field>
<field>
X-Mailer: Mozilla 3.01Gold (Win95; I)</field>
<field>
MIME-Version: 1.0</field>
<field>
To: lbj_ccsi@atl.mindspring.com</field>
<field>
Subject: Test Mail Again</field>
<field>
Content-Type: multipart/mixed; boundary="------------52E03A8932B4"</field>
</header>
<multipart>
<preamble>
This is a multi-part message in MIME format.
</preamble>
<body-part>
<header>
<field>
Content-Type: text/plain; charset=us-ascii</field>
<field>
Content-Transfer-Encoding: 7bit</field>
</header>
<body>
This is the message body.  A picture should also be displayed.
</body>
</body-part>
<body-part>
<header>
<field>
Content-Type: text/html; charset=us-ascii; name="Pull3.html"</field>
<field>
Content-Transfer-Encoding: 7bit</field>
<field>
Content-Disposition: inline; filename="Pull3.html"</field>
<field>
Content-Base: "file:///E|/ChckFree/Html/Pull3.html"</field>
</header>
<body>
&lt;BASE HREF="file:///E|/ChckFree/Html/Pull3.html"&gt;

&lt;HTML&gt;
&lt;HEAD&gt;
&lt;TITLE&gt;Client Pull Rolling Page Demo&gt;&lt;/TITLE&gt;

&lt;META HTTP-EQUIV="REFRESH" CONTENT="10; URL=Pull1.html"&gt;

&lt;/HEAD&gt;

&lt;BODY&gt;
This is Page 3 of my rolling web page demo.
&lt;/BODY&gt;

&lt;/HTML&gt;
</body>
</body-part>
<body-part>
<header>
<field>
Content-Type: image/gif; name="WWWIcon.gif"</field>
<field>
Content-Transfer-Encoding: base64</field>
<field>
Content-Disposition: inline; filename="WWWIcon.gif"</field>
</header>
<body>
R0lGODdhPAA8AOYAAP///+/v7+fv7+/39/f//9be3r3O1rXGzsbW3qW1va29xoyltXOMnGuE
lClSa4ScrXuUpUJje1p7lFJzjEprhDlacxhCYylScyFKaxA5WggxUgApSt7n773GzpytvZSl
tWuEnM7W3sbO1q29zqW1xoycrYSUpWN7lFpzjFJrhEJjhDlaezFScylKayFCYwgxWgApUtbe
5yFCaxg5YxAxWggpUpSlvXOEnFpzlFJrjEpjhEJaezlSc3uMpYyctYSUrb3G1sbO3qWtvbW9
zq21xufn79bW3t7e5wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAACwAAAAAPAA8AAAH/4AAgoOEhYaHiImKi4yNjo+QkZKTlJWW
l5iZmpucnZ6foKGinSIfJxEtMzMtEScfIqOKRwsUMy4YLQ4sLA4tGC4zFAtHsYQBCysWLSwr
KhQ5Eyg5FCorLC0WKwsBxUAoyhU6KA0MEA8lJQ8QDCcoOhXYKECjQhUyLDo4DA8fQkRDAIcQ
EfLhAQMcOljIqCAklI0WLVZMAPHAA5EDBoKE2BgCgYEDRDw8ADFhBUQbnzxgwLACBQMfCYYA
QRCiQIybMYx0NDAkgQ8GKFas9NCpAwsLLChQ9KAAowiaHDcGEfFRgUgQFI6y6LBJQA4aLnjk
APHjgwcSRBSMGLI24AgFRP9IePjwA0SOCi5o5BCgqQQMsEkl3Ohh4seDc+gSJz5sAsKNE1nz
wiiRKUYLGBoy3Nuhg0KKHNAmiB6NYjRoCjp2KMygAUaLGJj8woDxIoMFGRBz6drFu7eDXhAx
WMjwYvZkTCuMY35BI4Pz59CjR6fxorXxFZg4bNgxCIYMQydqwALwQ0OCQkVgROjuQtOGE4Nc
cBc0/gcMDoKEwBg/HgAM+ILIpwkLJgyyA4AAjCDICBsMYsAG+CU4CIEGInjJDkQJckKBABjA
oQHtCaKdgQZmCMCGmphgwCAmKNghgvMJwoKBEarIoouYmBDCIAusCMACMaZAiJCC0OCjjjz6
iIn/j/gpaQINDgIQoZIbLCAIkx1usmOWWwKQQoOC7OjjliFswOGWK3a5iYUA7LCBkj+q+WCM
GoJCJwA0bIAjADbyuMGMhNypyYOFbGBmoBwKYoKhhBDqyQgwDDAIB7PdQAgLlg5ywmySLhjp
IASkuN8gIsy2XnenChLBbP3ZB+clshlXgwYvMEcDdbXmquutuGpQg3LHXWKZcbRmMIMFFriA
7LLMLqusBTMQZ51rsMU2Ww21WYBLb7v89hu318gw3Au/BouJABRgRoO2LFSwQwTwxiuvvDtU
wIJwNLRGAV+adNCCBjTMgAEzEajQGQUIJ5ywDipEYC8GM+TbAleceDBDkmbKtPuuDgcrzHEE
O1iDDWszmMjJBxcb68IyzKyww7sgh2zNNcCQ/EEoHvxb27G4QbQbcC2IG211LZj8SQcpYEZr
c6ocC60qGVCnQWspUDyKACVcIOvUXHNdLgwXlMBvMQDEUMIOwKYNww4lVEt2IQaUgAILM8w2
AwsolPDq23z37fffgAcu+OCEF2744YgnXnggADs=</body>
</body-part>
<epilogue>


</epilogue>
</multipart>
</message>
