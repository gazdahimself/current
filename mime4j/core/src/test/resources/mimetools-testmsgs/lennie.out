Return-Path: <lbj_ccsi@atl.mindspring.com>
Received: from brickbat8.mindspring.com (brickbat8.mindspring.com [207.69.200.11])
        by camel10.mindspring.com (8.8.5/8.8.5) with ESMTP id GAA27894
        for <lbj_ccsi@atl.mindspring.com>; Fri, 25 Jul 1997 06:58:07 -0400 (EDT)
Received: from lennie (user-2k7i8oq.dialup.mindspring.com [168.121.35.26])
        by brickbat8.mindspring.com (8.8.5/8.8.5) with SMTP id GAA22488
        for <lbj_ccsi@atl.mindspring.com>; Fri, 25 Jul 1997 06:58:05 -0400 (EDT)
Message-ID: <33D89532.29EA@atl.mindspring.com>
Date: Fri, 25 Jul 1997 06:59:46 -0500
From: Lennie Jarratt <lbj_ccsi@mindspring.com>
Reply-To: lbj_ccsi@mindspring.com
Organization: Custom Computer Services Inc.
X-Mailer: Mozilla 3.01Gold (Win95; I)
MIME-Version: 1.0
To: lbj_ccsi@atl.mindspring.com
Subject: Test Mail Again
Content-Type: multipart/mixed; boundary="------------52E03A8932B4"

This is a multi-part message in MIME format.

--------------52E03A8932B4
Content-Type: text/plain; charset=us-ascii
Content-Transfer-Encoding: 7bit

This is the message body.  A picture should also be displayed.

--------------52E03A8932B4
Content-Type: text/html; charset=us-ascii; name="Pull3.html"
Content-Transfer-Encoding: 7bit
Content-Disposition: inline; filename="Pull3.html"
Content-Base: "file:///E|/ChckFree/Html/Pull3.html"

<BASE HREF="file:///E|/ChckFree/Html/Pull3.html">

<HTML>
<HEAD>
<TITLE>Client Pull Rolling Page Demo></TITLE>

<META HTTP-EQUIV="REFRESH" CONTENT="10; URL=Pull1.html">

</HEAD>

<BODY>
This is Page 3 of my rolling web page demo.
</BODY>

</HTML>

--------------52E03A8932B4
Content-Type: image/gif; name="WWWIcon.gif"
Content-Transfer-Encoding: base64
Content-Disposition: inline; filename="WWWIcon.gif"

R0lGODdhPAA8AOYAAP///+/v7+fv7+/39/f//9be3r3O1rXGzsbW3qW1va29xoyltXOMnGuElClS
a4ScrXuUpUJje1p7lFJzjEprhDlacxhCYylScyFKaxA5WggxUgApSt7n773GzpytvZSltWuEnM7W
3sbO1q29zqW1xoycrYSUpWN7lFpzjFJrhEJjhDlaezFScylKayFCYwgxWgApUtbe5yFCaxg5YxAx
WggpUpSlvXOEnFpzlFJrjEpjhEJaezlSc3uMpYyctYSUrb3G1sbO3qWtvbW9zq21xufn79bW3t7e
5wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACwA
AAAAPAA8AAAH/4AAgoOEhYaHiImKi4yNjo+QkZKTlJWWl5iZmpucnZ6foKGinSIfJxEtMzMtEScf
IqOKRwsUMy4YLQ4sLA4tGC4zFAtHsYQBCysWLSwrKhQ5Eyg5FCorLC0WKwsBxUAoyhU6KA0MEA8l
JQ8QDCcoOhXYKECjQhUyLDo4DA8fQkRDAIcQEfLhAQMcOljIqCAklI0WLVZMAPHAA5EDBoKE2BgC
gYEDRDw8ADFhBUQbnzxgwLACBQMfCYYAQRCiQIybMYx0NDAkgQ8GKFas9NCpAwsLLChQ9KAAowia
HDcGEfFRgUgQFI6y6LBJQA4aLnjkAPHjgwcSRBSMGLI24AgFRP9IePjwA0SOCi5o5BCgqQQMsEkl
3Ohh4seDc+gSJz5sAsKNE1nzwiiRKUYLGBoy3Nuhg0KKHNAmiB6NYjRoCjp2KMygAUaLGJj8woDx
IoMFGRBz6drFu7eDXhAxWMjwYvZkTCuMY35BI4Pz59CjR6fxorXxFZg4bNgxCIYMQydqwALwQ0OC
QkVgROjuQtOGE4NccBc0/gcMDoKEwBg/HgAM+ILIpwkLJgyyA4AAjCDICBsMYsAG+CU4CIEGInjJ
DkQJckKBABjAoQHtCaKdgQZmCMCGmphgwCAmKNghgvMJwoKBEarIoouYmBDCIAusCMACMaZAiJCC
0OCjjjz6iIn/j/gpaQINDgIQoZIbLCAIkx1usmOWWwKQQoOC7OjjliFswOGWK3a5iYUA7LCBkj+q
+WCMGoJCJwA0bIAjADbyuMGMhNypyYOFbGBmoBwKYoKhhBDqyQgwDDAIB7PdQAgLlg5ywmySLhjp
IASkuN8gIsy2XnenChLBbP3ZB+clshlXgwYvMEcDdbXmquutuGpQg3LHXWKZcbRmMIMFFriA7LLM
LqusBTMQZ51rsMU2Ww21WYBLb7v89hu318gw3Au/BouJABRgRoO2LFSwQwTwxiuvvDtUwIJwNLRG
AV+adNCCBjTMgAEzEajQGQUIJ5ywDipEYC8GM+TbAleceDBDkmbKtPuuDgcrzHEEO1iDDWszmMjJ
Bxcb68IyzKyww7sgh2zNNcCQ/EEoHvxb27G4QbQbcC2IG211LZj8SQcpYEZrc6ocC60qGVCnQWsp
UDyKACVcIOvUXHNdLgwXlMBvMQDEUMIOwKYNww4lVEt2IQaUgAILM8w2AwsolPDq23z37fffgAcu
+OCEF2744YgnXnggADs=

--------------52E03A8932B4--


